scripts/benchmark-plot.py

```diff
@@ -1,37 +1,46 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import argparse
 import glob
 import json
 import matplotlib.pyplot as plt
-import sys
+import numpy as np
 
 parser = argparse.ArgumentParser()
-parser.add_argument(
-    "-o", "--output", help="Save image to the given filename."
-)
+parser.add_argument("-o", "--output", help="Save image to the given filename.")
 
 args = parser.parse_args()
 
 commands = None
 data = []
 inputs = []
 
-for filename in glob.glob("scripts/tmp/benchmark-*.json"):
+for filename in sorted(glob.glob("scripts/tmp/benchmark-*.json")):
     with open(filename) as f:
         results = json.load(f)["results"]
     commands = [b["command"] for b in results]
     data.append([b["mean"] for b in results])
-    inputs.append(results[0]["parameters"]["input"])
 
-data = map(list, zip(*data))
-for (times, command) in zip(data, commands):
-    plt.plot(inputs, times, label=command)
+    input = results[0]["parameters"]["input"]
+    binary_size = results[0]["parameters"]["size"]
+    symbol_queries = results[0]["parameters"]["symbol_queries"]
+    inputs.append(f"{input}\n({binary_size}, {symbol_queries} qrs)")
```

Comment:
```suggestion
    inputs.append(f"{input}\n({binary_size}B,\n{symbol_queries} queries)")
```
"qrs" is a bit obscure.